src/unitxt/metrics.py

```diff
@@ -80,6 +80,14 @@ def new_random_generator():
     def disable_confidence_interval_calculation(self):
         self.n_resamples = None
 
+    def disable_confidence_interval_calculation_return_n_resamples(self):
```

Comment:
maybe just return `self.n_resamples` from the `disable_confidence_interval_calculation` method?
that shouldn't affect the existing code at all, no?